### PR TITLE
fill out schedule for final weeks

### LINKED ISSUE #30: set schedule for final weeks
* line up speaker(s?)
* find time for @PeterBell to come talk about professional presence
    * have students bring in resume for review
* check if @dbrizan has other topics he would 
* cover JS + AJAX
* is April 27 still a good end date?
* startup visit?

### PR DESCRIPTION
**Pull request by afeld**
Closes #30.

/cc @cuny-nytech/students @teddbrown1 @PeterBell @dbrizan

### CODE REVIEW

File: schedule.md [hunk 1]
```diff
@@ -220,11 +220,25 @@ TBD
...
+Prepare resume
 
 ## Apr 27
 
-* Final project demo (possibly at a meetup instead?)
+JS fundamentals
+
+## May 4
+
+* Mock interviews
```
Comment (afeld) [inline schedule.md]: @PeterBell et al: Mind helping me find 1-3 outside people to come in and do this? Alternatively, we could go to someone's office if it's easier for them.

**Comment by afeld** [comment]
Merge early, iterate often.

File: schedule.md [hunk 2]
```diff
@@ -220,11 +220,28 @@ TBD
...
 ## Apr 27
 
-* Final project demo (possibly at a meetup instead?)
+* JS/jQuery fundamentals
+* Resume review
+
+## May 4
+
+* Mock interviews
+* Professional Presesence talk w/ Peter
```
Comment (afeld) [inline schedule.md]: @PeterBell Are you planning to come in person for this, or will it be over video?

File: schedule.md [hunk 3]
```diff
@@ -220,11 +220,28 @@ TBD
...
+* Final project demo prep
+
+## TBD
+
+* Final project demo at a meetup
 * Farewell dinner
```
Comment (afeld) [inline schedule.md]: Will send a Doodle to find availability for these.

File: schedule.md (hunk 2 above)
Comment (PeterBell) [inline schedule.md]: @afled I can do this one in person!

**Pull request merged**